index.html

```diff
@@ -46,8 +47,7 @@ <h1 class="greyfont">Bomb Defusal Manual - Unofficial Translations</h1>
                                                       href="/PDF/KeepTalkingAndNobodyExplodes-BombDefusalManual-v1-RU.pdf">PDF</a><br>
     <a class="button" href="/spanish/">Spanish</a><a class="button pdf"
                                                      href="/PDF/KeepTalkingAndNobodyExplodes-BombDefusalManual-v1-ES.pdf">PDF</a><br>
-    <a class="button" href="/swedish/">Swedish</a><a class="button pdf"
-                                                     href="/PDF/KeepTalkingAndNobodyExplodes-BombDefusalManual-v1-SV.pdf">PDF</a><br>
+    <a class="button" href="/swedish/" style="width:305px">Swedish</a><br>
```

Comment:
Why the Swedish PDF button is now missing?

Reply:
Since Swedish includes a subsite with all of the Web-views/PDFs, I thought it might be better to show it in a unique way rather than blend it with all the others. In this way, I believe that the user understands that there's something different here, yet it does not change the entire pattern of the main web page

Reply:
Would appreciate if a similar thing can be made with the Hebrew version: https://hebrewbombmanual.github.io/